### DISCUSSION TITLE
Use treesitter scopes in themes

### DISCUSSION THREAD

**Discussion by supersurviveur** [helix-editor/helix#11147] [Q&A] [answered]
Is there a way to use tree sitter scopes in themes ? Not just the highlight name.
Thanks !

**Comment by supersurviveur** [comment]
Seems that label are assigned from `highlights.scm` files. Can these files be edited ? Can i use custom labels not present here https://docs.helix-editor.com/themes.html

**Comment by supersurviveur** [comment, marked as answer]
Finally it works !
Label can be set in `highlights.scm` files in the runtime dir (`/usr/lib/helix/runtime/` on my linux). No need to rebuild, they are loaded dynamically.
Then you can use these labels in your theme.

However I don't know if more complex labels should be added to helix directly, not in custom config.

**Reply by probablySophie** [reply]
I think I'm trying to do the same thing you were doing here.  Do you have an example .scm line or two that would explain how you did it?

Thank you

**Reply by supersurviveur** [reply]
Sure !
For example :
https://github.com/supersurviveur/dotfiles/blob/main/.config%2Fhelix%2Fruntime%2Fqueries%2F_jsx%2Fhighlights.scm#L36-L37

Then
https://github.com/supersurviveur/dotfiles/blob/main/.config%2Fhelix%2Fthemes%2Fmonokai_custom.toml#L13
